modules/crop/architecture.md

```diff
@@ -0,0 +1,47 @@
+# Architecture
+
+Diagram shows basic architecture of Crop.
+
+![Crop architecture](images/architecture.png)
+
+## Components
+
+### Entity
+- **Crop** Is a content entity that stores various informations related to the crop area.
+- **CropType** CropType is bundle for Crop entity.
+
+###Plugin
+- **CropEffect** Crop effect is an image style effect. He apply your crop area into image with information of Crop entity.
+
+This diagram shows how specifically Crop Image Effect (Manual crop) get informations of corresponding image ressource and apply different effect.
```

Comment:
Again. What are we trying to explain with this diagram? I don't think we need it. Sentence above explains that image effect crops image based on the crop entity. That should be enough.